### DISCUSSION TITLE
ShaderGalss won't work when dosbox is in full screen mode

### DISCUSSION THREAD

**Discussion by lordgeyik** [mausimus/ShaderGlass#73] [General]
I chose the clone mode for output, and the game which is running in full screen for input. When doing this, the shader glass window shrinks to the part of the tool bar only, and the shader has no effect on the game when I alt-tab back to it.

I also tried using glass mode, and using shader glass in full screen. The shader only affects windowded apps. As soon as I go fdullscreen on any app, the shader won't work.

I'm using up-to-date Windows 10.

**Comment by armentix** [comment]
Hi, i know it's late answer. I also crossed the path of this tool and was using it intensively. 
The only solution i found is using Windows Powertoys (free from Microsoft's store) that has a feature to set a window always on top which correct this problem (until the dev adds this option to the app itself).

**Comment by mausimus** [comment]
Either set DOSBox to use borderless fullscreen, or run it in a Window and fullscreen it through ShaderGlass (if you need the mouse, you might need to se Output -> Window -> Click-through, see the Manual).
